requirements/base.txt

```diff
@@ -1,2 +1,3 @@
 # .env file reading
-python-decouple==3.1
\ No newline at end of file
+python-decouple==3.1
+alembic==1.2.1
```

Comment:
Good! But please add that to requirements/web.txt instead.
Also add sync driver `psycopg2-binary`